src/workerd/api/global-scope.h

```diff
@@ -382,6 +382,35 @@ struct ExportedHandler {
   jsg::Optional<jsg::Ref<ExecutionContext>> getCtx() {
     return ctx.map([&](jsg::Ref<ExecutionContext>& p) { return p.addRef(); });
   }
+
+  template <typename T>
+  static jsg::LenientOptional<T> ADDREF(jsg::LenientOptional<T>& function, jsg::Lock& js) {
+    return function.map([&](T& a) { return a.addRef(js); });
+  }
+
+#define CLONE(x)                                                                                   \
```

Comment:
Defining this macro means that anything in the codebase which uses the identifier `CLONE` will now be broken.

Generally, there are two ways to solve this:
* Use a namespace prefix on the macro so that it is unlikely to collide, like `WORKERD_EXPORTED_HANDLER_CLONE`.
* `#undef` the macro when you're done using it. (This is best for cases like this, where the macro is only meant to be used locally.)

However, I would recommend just _not_ using a macro here. It doesn't seem worth it, the initializers are not that verbose:

```
.fetch = mapAddRef(js, fetch),
```